fastlane_core/spec/update_checker_spec.rb

```diff
@@ -59,6 +59,11 @@
         ENV["FASTLANE_SELF_CONTAINED"] = "true"
         expect(FastlaneCore::UpdateChecker.update_command).to eq("fastlane update_fastlane")
       end
+      
+      it "works with Mac app-installed fastlane" do
+        ENV["FASTLANE_SELF_CONTAINED"] = "false"
```

Comment:
Nice. One possible improvement... can you try it with:

```ruby
with_env_values('FASTLANE_SELF_CONTAINED' => 'true') do
  expect(FastlaneCore::UpdateChecker.update_command).to eq("the Fabric app. Launch the app and navigate to the fastlane tab to get the most recent version.")
end
```

That will make sure that the ENV value is reset after the block completes so that no state leaks from the test 👍